doc/BACKUP.md

```diff
@@ -0,0 +1,413 @@
+Backing Up Your C-Lightning Node
+================================
+
+Lightning Network channels get their scalability and privacy benefits
+from the very simple technique of *not telling anyone else about your
+in-channel activity*.
+This is in contrast to onchain payments, where you have to tell everyone
+about each and every payment and have it recorded on the blockchain,
+leading to scaling problems (you have to push data at everyone) and
+privacy problems (everyone knows every payment you were ever involved in).
+
+Unfortunately, this removes a property that onchain users are so used
+to, they react in surprise when learning about this removal.
+Your onchain activity is recorded in all archival fullnodes, so if you
+forget all your onchain activity because your storage got fried, you
+just go redownload the activity from the nearest archival fullnode.
+
+But in Lightning, since *you* are the only one storing all your
+financial information, you ***cannot*** recover this financial
+information anywhere else.
+
+This means that on Lightning, you **have to** responsibly back up your
+financial information yourself, using various processes and automation.
+
+The discussion below assumes that you know where you put your
+`$LIGHTNINGDIR`, and you know the directory structure within.
+By default your `$LIGHTNINGDIR` will be in `~/.lightning/${COIN}`.
+For example, if you are running `--mainnet`, it will be
+`~/.lightning/bitcoin`.
+
+`hsm_secret`
+------------
```

Comment:
Mix of `----` and `###` headings makes it more difficult to judge the overall structure. Please use `###` prefixes wherever possible.

Reply:
I mildly dislike the `###` prefixes since to me "more `#` = more heavier" i.e. `###` should map to `<h1>`, `#` should map to `<h3>`.  *shrug*.  Changed anyway.